debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/AlterTableParserListener.java

```diff
@@ -164,7 +164,7 @@ public void enterAlterByChangeColumn(MySqlParser.AlterByChangeColumnContext ctx)
             }
             else {
                 throw new ParsingException(null, "Trying to change column " + oldColumnName + " in "
```

Comment:
Let's log at trace level since these are DDL - 
```
LOG.trace("Trying to alter table " + tableId.toString()
    + ", which does not exist. Query: " + getText(ctx));
throw new ParsingException(null, "Trying to alter table " + tableId.toString()
    + ", which does not exist.");
```
Same change in the other places too (this and the other 2 classes)